aws_jobs/cyhy-data-extract.py

```diff
@@ -85,6 +91,28 @@ def flatten_datetime(in_datetime):
     return in_datetime.replace(hour=0, minute=0, second=0, microsecond=0)
 
 
+# All logging code is pulled from cyhy-core and tweaked down to this single use-case.
+# Since we are still running Python2 we cannot leverage some of the improvements
+# made in the logging library in later version.
+def setup_logging(debug_logging):
+    """Set up logging for the script."""
+    LOGGER_FORMAT = "%(asctime)-15s %(levelname)s %(name)s - %(message)s"
+    formatter = logging.Formatter(LOGGER_FORMAT)
+    formatter.converter = time.gmtime  # log times in UTC
+    root = logging.getLogger()
+    if debug_logging:
+        root.setLevel(logging.DEBUG)
+    else:
+        root.setLevel(DEFAULT_LOGGER_LEVEL)
+    file_handler = RotatingFileHandler(
+        LOG_FILE, maxBytes=pow(1024, 2) * 128, backupCount=9
```

Comment:
It would be nice to make the value of `backupCount` into a constant that we declare up with `DEFAULT_LOGGER_LEVEL`, in the spirit of bringing all of our magic numbers into the light.

Reply:
Good suggestion, resolved in 01234d3